### PR TITLE
Issue #11214: Update for EmptyLineSeparator

### PR DESCRIPTION
**Pull request by Shaman20**
Fixes Issue for https://github.com/checkstyle/checkstyle/issues/11214

### CODE REVIEW

**Comment by Shaman20** [comment]
image

Is this error related to my project build because yesterday the build was successful but today after editing a InputEmptyLineSeparator.java this error seemed to appear. I'm not sure if it was from the edit . Please help

**Comment by Shaman20** [comment]
image

image

Can you please tell me how do I reduce the above comment size?

**Comment by romani** [comment]
You can drop "should be" 
You can use `// violation above ...` you can find bunch of examples in our sources.

You can rename any name to make it shorter.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public int compareTo2(Object aObject) // empty line - ok
      * @param result
      * @return
      */
-    public static <T> Callable<T> callable(Runnable task, T result) // violation
+    public static <T> Callable<T> callable(Runnable task, T result) // violation ''METHOD_DEF' should be separated from previous line.'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java:75]: You can remove parameter `Runnable task,` to shorten line.

`public static <T> Callable<T> clb(T r)`

**Comment by Shaman20** [comment]
image

@romani I am not sure why is this error occurring could you please help?

**Comment by nick-mancuso** [comment]
> image
> 
> @romani I am not sure why is this error occurring could you please help?

Ant task problems (checkstyle failures) will be above `BUILD FAILURE` line (out of screenshot), please read them carefully.

**Comment by Shaman20** [comment]
> > image
> > @romani I am not sure why is this error occurring could you please help?
> 
> Ant task problems (checkstyle failures) will be above `BUILD FAILURE` line (out of screenshot), please read them carefully.

Sure I'll check them out

**Comment by Shaman20** [comment]
@romani 
image


image

On getting the above error added this 
image

But still getting the error
image

Could you please guide me further on how to resolve this error?

**Comment by Shaman20** [comment]
image

image

image

image

@romani @nick-mancuso 
I am not sure on how to shorten these messages and keep it meaningful. Could you please guide me further

**Comment by nick-mancuso** [comment]
See https://github.com/checkstyle/checkstyle/pull/12056#issuecomment-1214275618, remember to always search through codebase for answers.

**Comment by Shaman20** [comment]
@romani @nick-mancuso I had a look at other PR's and I looked at this 
image

I didn't understand this part Could you please tell if it is the line number?

**Comment by Shaman20** [comment]
@romani @nick-mancuso 
I am not sure how to resolve this could you please guide me further?

image

**Comment by rnveach** [comment]
@Shaman20 Do you still need assistance?

**Comment by rnveach** [comment]
Since contact seems to be lost, I rebased the branch and fixed the violations.

**Comment by rnveach** [comment]
It looks like other input files may be missing this update, so I am putting this back in draft.

**Comment by rnveach** [comment]
I made some more changes. It doesn't complete the check but it is a lot more.

I found the following changes assisted in determining the text to enter:
````
diff --git a/src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java b/src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java
index 157e654..ff7c50f 100644
--- a/src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java
@@ -407,7 +407,9 @@
         assertWithMessage("Violation lines for %s differ.", file)
                 .that(actualViolationLines)
                 .isEqualTo(expectedViolationLines);
+        System.out.println(new Exception().getStackTrace()[2].getMethodName());
         for (int index = 0; index < actualViolations.size(); index++) {
+            System.out.println(index + " - " + actualViolations.get(index));
             assertWithMessage("Actual and expected violations differ.")
                     .that(actualViolations.get(index))
                     .matches(testInputViolations.get(index).toRegex());
````

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
There supposed to be removal from suppression

**Comment by rnveach** [comment]
@romani Doesn't the whole check have to be done? As I said at https://github.com/checkstyle/checkstyle/pull/12056#issuecomment-1309640723 , this doesn't complete the whole check. The original PR did not have the entire check.

**Comment by romani** [comment]
Ok issue stay open, new PR to finish is welcome

**Pull request merged**